_inc/lib/core-api/wpcom-endpoints/publicize-services.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * Publicize: List Publicize Services
+ *
+ * [
+ *   { # Service Object. See schema for more detail.
+ *     name:  (string) Service slug
+ *     label: (string) Human readable label for the Service
+ *     url:   (string) Connect URL
+ *   },
+ *   ...
+ * ]
+ *
+ * @since 6.8
+ */
+class WPCOM_REST_API_V2_Endpoint_List_Publicize_Services extends WP_REST_Controller {
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'publicize/services';
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Called automatically on `rest_api_init()`.
+	 */
+	public function register_routes() {
+		register_rest_route( $this->namespace, '/' . $this->rest_base, array(
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_items' ),
+				'permission_callback' => array( $this, 'get_items_permission_check' ),
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+	}
+
+	public function get_item_schema() {
+		$schema = array(
+			'$schema' => 'http://json-schema.org/draft-04/schema#',
+			'title' => 'jetpack-publicize-service',
+			'type' => 'object',
+			'properties' => array(
+				'name' => array(
+					'description' => __( 'Alphanumeric identifier for the Publicize Service', 'jetpack' ),
```

Comment:
A language nit: "for the Publicize Service" looks ambiguous - it's not clear whether this means "for Publicize, itself", or "for a service that Publicize supports".

Reply:
Yeah - I can see the confusion. Since this is the "publicize/services" endpoint, I think it's OK, though.

Happy to change it if you have a suggestion, of course :)